src/VirtualClient/VirtualClient.Main/profiles/PERF-GPU-SPECVIEW-AMD.json

```diff
@@ -7,7 +7,8 @@
   },
   "Parameters": {
     "GpuModel": "v620",
-    "Viewset": "3dsmax,catia"
+    "Viewset": "3dsmax,catia",
```

Comment:
I remember previously there were more viewsets, is there a reason to remove them?

Reply:
There are in total 8 view sets. Running all eight viewsets can take hours on a small GPU. Thus I only included two viewsets as the default argument.  Users can see the full list of viewsets from the doc.